examples/templates/do-linux/main.tf

```diff
@@ -96,6 +96,12 @@ data "coder_workspace" "me" {}
 resource "coder_agent" "dev" {
   os   = "linux"
   arch = "amd64"
+  env = {
+    GIT_AUTHOR_NAME = "${data.coder_workspace.me.owner}"
+    GIT_COMMITTER_NAME = "${data.coder_workspace.me.owner}"
+    GIT_AUTHOR_EMAIL = "${data.coder_workspace.me.owner_email}"
+    GIT_COMMITTER_EMAIL = "${data.coder_workspace.me.owner_email}"
+  }
```

Comment:
I think we should add this to our docs (which you can do in this PR too), but we shouldn't do it in all templates. I think it could give the perception that this is mandatory when it certainly isn't.

Reply:
Makes sense. In that case I'd lean towards including it in the `docker` and `docker-code-server` templates, since those are the ones that are most likely to be used for a "quickstart", and removing it from the others. I'll add documentation as well.

Reply:
I agree with that. Makes sense to me!